lib/src/backend/declarer.dart

```diff
@@ -127,7 +133,8 @@ class Declarer {
       skip,
       Map<String, dynamic> onPlatform,
       tags,
-      int retry}) {
+      int retry,
+      bool solo = false}) {
```

Comment:
Consider marking the param `@Deprecated` – so any usage **screams** to the user ... also caught if folks check for lints in their CI environments...

Reply:
Good call.